domains/dave9123.json

```diff
@@ -0,0 +1,10 @@
+
+{
+  "owner": {
+    "username": "dave9123",
+    "discord": "Dave9123#0757",
```

Comment:
i would like to see an email here.

if you dont have or dont wanna leak, fill in the email with an empty string like this: `{"email": ""}`

Reply:
Where should I place?